### ISSUE TITLE
Latest Version - Asking to Update

### ISSUE THREAD

**Issue by he3als** [Nilsen84/lunar-client-qt#50]
I am on the latest version, yet it is asking me to update.

![image](https://user-images.githubusercontent.com/65787561/160872784-2f7f31a8-4c72-4cd4-93c7-7813634084e1.png)

**Comment by he3als** [comment]
Downloaded the latest from releases again and it persists. https://github.com/Nilsen84/lunar-client-qt/releases/download/v1.2.4/windows.zip

**Comment by ceilingfans** [comment]
I tried this on macOS 12.3 and this issue happens too, a fix should be implemented sooner or later. In the meantime you can just ignore the window

**Comment by he3als** [comment]
Yeah, I presume it is something simple, I might make a PR for it.

**Comment by ceilingfans** [comment]
Looking further into this, the way lcqt checks for the current version is using this snippet of code
https://github.com/Nilsen84/lunar-client-qt/blob/9a86586d9b35b3d249ff875f24ee3ab28d18a8ed/CMakeLists.txt#L30

When running the git command which is `git describe --match 'v[0-9]*'`, I get the output of `v1.2.3-48-g19732d8` so that is why it is detecting v1.2.3 instead of v1.2.4

**Comment by Nilsen84** [comment]
Should be fixed in the latest release, it was caused by the incredibly fragile ci pipeline
